Kajabi/sage-lib
# feat(icons): add new icons

## Description

Implement two new icons: columns and handle-2-vertical


## Screenshots

|  Before  |  After  |
|--------|--------|
| ![before-icons](https://user-images.githubusercontent.com/24237393/141372586-c4df5e55-ab27-4259-9688-5847981bb70a.png) | ![after-icons](https://user-images.githubusercontent.com/24237393/141372657-8498bae5-399b-4c66-b039-9d8a9db2a05f.png) |

## Testing in `sage-lib`

1. Navigate to http://localhost:4000/pages/foundations/icon
2. Verify icons are present and no other icons are missing
3. Navigate to http://localhost:4100/?path=/story/sage-icon--default
4. Verify icons are present and no other icons are missing

## Testing in `kajabi-products`
1. (**LOW**) New icons in `sage-lib`; no testing in `kajabi-products` necessary
 
## Related
Closes #1021
